ref/feature/openidConnectServer-1.0.adoc

```diff
@@ -1,12 +1,14 @@
 = OpenID Connect Provider 1.0
-:linkcss: 
+:linkcss:
 :page-layout: feature
-:nofooter: 
+:nofooter:
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=description]
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=enable]
 
+include::./gen/openidConnectServer-1.0./examples.adoc[]
```

Comment:
I think this should be:
`include::./openidConnectServer/examples.adoc[]`

(Never done this before, but going off the example from https://raw.githubusercontent.com/OpenLiberty/docs/master/ref/feature/jdbc-4.0.adoc.)

Also, do you need to add the description file too?

Reply:
Yes. Need to add the description file but need to discuss before I do so as not sure.